molgenis-ui-settings/src/main/resources/templates/view-settings.ftl

```diff
@@ -0,0 +1,24 @@
+<#include "resource-macros.ftl">
+<#include "molgenis-header.ftl">
+<#include "molgenis-footer.ftl">
+
+<#assign js = []>
+<#assign css = ["settings/app.css"]>
+<#assign version = 2>
+
+<@header css js version/>
+    <div id="app"></div>
+
+    <script type="text/javascript">
+        window.__INITIAL_STATE__ = {
+            baseUrl: '${baseUrl}',
+            lng: '${lng}',
+            fallbackLng: '${fallbackLng}',
+            isSuperUser: ${isSuperUser?c}
+        }
+    </script>
+
+    <script type=text/javascript src="<@resource_href "/js/settings/manifest.js"/>"></script>
```

Comment:
for some reason i am getting 404 errors on fetching 'manifest' and 'verdor' files, maybe some problem/typo in prod.conf ?